config/settings/base.py

```diff
@@ -364,7 +364,7 @@
 API_INSEE_CONSUMER_KEY = os.getenv("API_INSEE_CONSUMER_KEY")
 API_INSEE_CONSUMER_SECRET = os.getenv("API_INSEE_CONSUMER_SECRET")
 # https://api.insee.fr/catalogue/site/themes/wso2/subthemes/insee/pages/item-info.jag?name=Sirene&version=V3&provider=insee
-API_INSEE_SIRENE_BASE_URL = f"{API_INSEE_BASE_URL}/entreprises/sirene/V3"
+API_INSEE_SIRENE_BASE_URL = f"{API_INSEE_BASE_URL}/entreprises/sirene/V3.11"
```

Comment:
De ce que je comprend de l'encart on est pas obligé de spécifier la version donc autant se simplifier la tâche je pense :
```suggestion
API_INSEE_SIRENE_BASE_URL = f"{API_INSEE_BASE_URL}/entreprises/sirene"
```
```
>>> settings.API_INSEE_SIRENE_BASE_URL = "https://api.insee.fr/entreprises/sirene"
>>> api_entreprise.etablissement_get_or_error("13003013300016")
{"message": "HTTP Request: POST https://api.insee.fr/token \"HTTP/1.1 200 OK\"", "logger.name": "httpx", "logger.thread_name": "MainThread", "logger.method_name": "_send_single_request", "date": "2024-05-13T10:49:33.370788+00:00", "status": "INFO"}
{"message": "HTTP Request: GET https://api.insee.fr/entreprises/sirene/siret/13003013300016?date=2024-05-13 \"HTTP/1.1 200 \"", "logger.name": "httpx", "logger.thread_name": "MainThread", "logger.method_name": "_send_single_request", "date": "2024-05-13T10:49:33.435719+00:00", "status": "INFO"}
(Etablissement(name="PLATEFORME DE L'INCLUSION", address_line_1='127 RUE DE GRENELLE', address_line_2=None, post_code='75007', city='PARIS', department='75', is_head_office=True, is_closed=False), None)

```

Reply:
J'ai peur que `/entreprises/sirene` ne soit un jour équivalent à `/entreprises/sirene/v4` :grimacing:

Reply:
Bon, normalement la v4 n'arrivera pas avant 2025 et sera sûrement dans un premier temps accessible en opt-in.
Je vais suivre ta recommandation et il faut qu'on trouve le moyen d'être tenu au courant des évolutions de l'API pour anticiper la prochaine fois :)

Reply:
> J'ai peur que `/entreprises/sirene` ne soit un jour équivalent à `/entreprises/sirene/v4` 😬

Certes, mais :
1. La V4 pourrais être compatible avec ce qu'on utilise de la V3, sachant qu'on utilise pas grand chose et que des trucs vraiment basiques.
2. Si en fait ça casse on le verra sûrement plus facilement (`KeyError` je pense) que des 404.

> [...] il faut qu'on trouve le moyen d'être tenu au courant des évolutions de l'API pour anticiper la prochaine fois :)

On est d'accord :).